tests/testthat/_snaps/groups.md

```diff
@@ -0,0 +1,42 @@
+# get_group_content() successfully gets the content for multiple groups
+
+    Code
+      get_group_content(client, groups_df)
+    Condition
+      Warning:
+      Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
+      i Please use `"guid"` instead of `.data$guid`
+      Warning:
+      Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
+      i Please use `"name"` instead of `.data$name`
```

Comment:
These warnings don't appear when I use the function from the command line. As far as I can tell, there are no other good options here:

- Just using `guid` with no qualifications gives lint errors
- Using `"guid"` fixes lint errors but changes the names of the resulting data frame to `"guid"` with quotation marks(!)